src/components/candidates/Candidate.vue

```diff
@@ -60,6 +33,32 @@ async function getCandidateData() {
     console.log(e);
   }
 }
+
+const { getResultsForCandidate } = useResultsStore();
+const candidateResults = ref<Result[]>([]);
+async function getResultsForCandidateData() {
+  try {
+    const { data } = await getResultsForCandidate(params.id);
+    candidateResults.value = data;
+  } catch (e) {
+    console.log(e);
+  }
+}
+getResultsForCandidateData();
+
+const formType = ref("put");
+const { getCandidateById, deleteCandidateById, editCandidate } =
+  useCandidateStore();
+
+let candidateInit = {
```

Comment:
"let" isn't needed since it never changes. It might look better if you create a separate function to reset a candidate instead of using this object since it can potentialy be changed somewhere